### PR TITLE
#35/bruteforce/generate schedule

### PR DESCRIPTION
**Pull request by Askir**
adds a schedule page as discussed in #35

### CODE REVIEW

File: app/models/event.rb [hunk 1]
```diff
@@ -45,4 +49,40 @@ def end_after_start
       errors.add(:enddate, "must be after startdate.")
     end
   end
+
+  # Everything below this is leagues only code and will be moved to Leagues.rb once there is an actual option to create Leagues AND Tourneys, etc.
+
+  def add_test_teams
+    self.max_teams.times do |index|
+      self.teams << Team.new(name: "Team #{index}", private: false)
```
Comment (erksch) [inline app/models/event.rb]: `self.` is not actually neccessary here and you do not use it anywhere else, so please maintain constistent code style.

File: app/models/event.rb [hunk 2]
```diff
@@ -35,7 +39,7 @@ def self.types
   scope :active, -> { where('deadline >= ?', Date.current) }
 
   def duration
-    return if enddate.blank? || startdate.blank?
+    return 10 if enddate.blank? || startdate.blank?
```
Comment (erksch) [inline app/models/event.rb]: Please remove the 10 here.

**Review by erksch — APPROVED** [review]

**Comment by manner** [comment]
index method in MatchesController is missing. Please fix that.

**Comment by Askir** [comment]
Intended behaviour there exists a schedule page for each event now

**Pull request merged**